test/unit/services/workers/test-send-advance-claim-evidence-reminder-notification.js

```diff
@@ -9,15 +9,20 @@ const EMAIL_ADDRESS = 'test@test.com'
 const REFERENCE = '1234567'
 const ELIGIBILITY_ID = 4321
 const CLAIM_ID = 1234
+var firstName = 'Joe'
 
 const dateOfJourney = new Date('2016-02-01')
 const dateOfJourneyString = '1 February 2016'
 
 var stubGetClaim = sinon.stub().resolves({DateOfJourney: dateOfJourney})
 var stubSendNotification = sinon.stub().resolves()
+var getFirstNameByClaimId = sinon.stub().resolves({
```

Comment:
Sorry for being nit-picky, but can you change the name of this to `stubGetFirstNameByClaimId` so that it follows the naming convention of other stubs?  I'll not bother adding a comment for all the other instances of this stub in other test files, but can you update the variable name if it occurs elsewhere?